packages/backend/src/machine-utils.ts

```diff
@@ -174,6 +174,11 @@ export function isLinux(): boolean {
   return linux;
 }
 
+const darwin = os.platform() === 'darwin';
+export function isMac(): boolean {
+  return darwin;
+}
+
```

Comment:
You could use

```ts
import { env } from '@podman-desktop/api'
env.isMac
```

Reply:
opened up: https://github.com/containers/podman-desktop-extension-bootc/issues/881

Reply:
updated the code as well :)